### PR TITLE
Fix oppia#4760 In Terms of Service the website should be displayed as link and should be clickable

### LINKED ISSUE #4760: In Terms of Service the website should be displayed as link and should be clickable
**Describe the bug**
In Terms of Service the website should be displayed as link and should be clickable

**To Reproduce**
1. Go to Menu - Help
2. Click on Terms of Service
3. In the 1st paragraph look at the website link 

**Expected behavior**
The link https://www.oppia.org should be displayed as a link and it should be clickable.
The sentence ' Thanks for your interest in Oppia Foundation Inc., a nonprofit that operates the website located at https://www.oppia.org (the Website).'

Website is mentioned twice in the statement.

**Demonstration**
If applicable, add screenshots or videos to help explain your problem.
Screenshot_20221124-143700

**Environment**
Device/emulator being used: Infinix smart 5A
Android or SDK version (e.g. Android 5 or SDK 21): Android version 11
App version (you can get this through system app settings or via the admin controls menu in-app): 0.10-beta-5e64fae55e

### PR DESCRIPTION
**Pull request by Tejas-67**
## Explanation
Fix #4760 In Terms of Service the website should be displayed as link and should be clickable

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- For PRs introducing new UI elements or color changes, both light and dark mode screenshots must be included
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: gradle.properties
Comment (adhiamboperes) [inline gradle.properties:1]: Please remove this from the PR

File: app/src/main/res/values/terms_of_service.xml [hunk 1]
```diff
@@ -10,14 +10,14 @@
       Visitors or registered users to this website are referred to as "Users" or
       an individual "User".
     </p>
-
+    <br>
```
Comment (adhiamboperes) [inline app/src/main/res/values/terms_of_service.xml:13]: Could you explain why you modified this file? Ditto the privacy_policy.xml above.

File: .gitignore
Comment (adhiamboperes) [inline .gitignore:1]: Remove this file

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @Tejas-67!

I will take a deep dive into your solution later, but for now, there are PR-specific items for you to fix. Please see this [wiki section](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR) on submitting PRs, and also take a look at the Essential checklist on this PR.
1. branch name should not be develop
2. android studio, idea or git related files that change configs should be excluded from the PR.
3. PR description will include an explanation of the fix that you are implemenating(What you changed and why)
4. Please sign the CLA as instructed

Overall, please check out your code to a new branch not called develop, and create a new PR to replace this one.

File: gradle.properties
Comment (Tejas-67) [reply gradle.properties:1]: okay!

File: app/src/main/res/values/terms_of_service.xml (hunk 1 above)
Comment (Tejas-67) [reply app/src/main/res/values/terms_of_service.xml:13]: The "br" tags were added to ensure the correct space formatting between the different sections of the pages.

File: .gitignore
Comment (Tejas-67) [reply .gitignore:1]: okay

**Comment by adhiamboperes** [comment]
@Tejas-67, I have noticed you have not created a replacement PR for this. Are you experiencing any issues?

**Comment by Tejas-67** [comment]
@adhiamboperes  hey, sorry for the delay. I am having my exams hence will be raising a PR in some days.

**Pull request closed without merging**